docker-compose.yml

```diff
@@ -60,31 +60,6 @@ services:
       - './packages:/var/www/html/wp-content/plugins/packages'
       - './templates:/var/www/templates'
 
-  test_wordpress:
```

Comment:
Nice! Happy to see this cleanup! We could probably also remove the [test database initialization in the DB service](https://github.com/mailpoet/mailpoet/blob/74e1616e15de23e2666ba1dcfb0c9e76e5e0ba1d/docker-compose.yml#L15). AFAIK it was used only for the test site.